src/AuthProps.ts

```diff
@@ -0,0 +1,85 @@
+import type { User, UserManager, UserManagerSettings } from "oidc-client-ts";
+
+interface AuthProviderPropsBase extends UserManagerSettings {
```

Comment:
Please keep `AuthProviderProps` where they are, only add userManager. Try to avoid refactoring when adding a simple feature.

Reply:
it was actually not so simple - the reason for this was to add type safety for using usermanager vs the usermanagersettings (redirect_uri, client_id, etc.) and to avoid cluttering up the main provider with multiple interface declarations

however, since its a subjective choice and this is your lib, ill revert it.

Reply:
nicely done!